specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/examples/inference/createOrUpdateAMLComputeOnlineEndpointDeployment.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "onlineEndpointName": "testendpoint",
+    "deploymentName": "testdeployment",
+    "api-version": "2020-09-01-preview",
+    "properties": {
+      "location": "westus2euap",
+      "tags": {},
+      "kind": "AMLCompute",
+      "sku": {
+        "name": "Standard_DS2_V2",
+        "tier": "Basic",
+        "capacity": 1
+      },
+      "properties": {
+        "description": "This is a test deployment",
+        "properties": {},
+        "modelId": "modelURL",
+        "codeConfiguration": {
+          "codeArtifactId": "testCodeArtifactId",
+          "command": [
+            "testCommand"
+          ]
+        },
+        "environmentId": "testEnvironmentId",
+        "skuScaleSettings": {
+          "minimum": 1,
+          "maximum": 5
+        },
+        "environmentOverrides": {
+          "environmentVariables": {
+            "envKey": "envValue"
+          }
+        },
+        "deploymentConfiguration": {
```

Comment:
remove

Reply:
need for discriminator for DeploymentConfiguration